app/main.py

```diff
@@ -924,6 +924,22 @@ def get_available_uberonids(query):
     return jsonify(result)
 
 
+@app.route("/simulation_ui_file/<identifier>")
+def simulation_ui_file(identifier):
+    results = process_results(dataset_search(create_pennsieve_identifier_query(identifier)))
+    results_json = json.loads(results)
+
+    try:
+        item = results_json["results"][0]
+        uri = item["s3uri"]
+        path = item["abi-simulation-file"][0]["dataset"]["path"]
+        key = f"{uri}files/{path}".replace(f"s3://{Config.S3_BUCKET_NAME}/", "")
+
+        return json.loads(direct_download_url(key))
```

Comment:
We should be using the `jsonify` function here as in line 924 to properly set headers.

Reply:
Agreed. Consider it done.

Otherwise, are we ok with the fact that `jsonify()` is not being used throughout the API?

Reply:
Fixed in 2ac1a75.